Makefile

```diff
@@ -5,10 +5,12 @@ ARCHIVE=distrobuilder-$(VERSION).tar
 
 default:
 	gofmt -s -w .
-	go get -t -v -d ./...
 	go install -v ./...
 	@echo "distrobuilder built successfully"
 
+update-gomod:
+	go get -t -v -d -u ./...
```

Comment:
This should do a tidy run, no?

Reply:
`go get` will actually update `go.mod` and `go.sum`. Running `go mod tidy` afterwards won't change anything.

Reply:
I thought `go mod tidy` would remove duplicate entries, no?

Reply:
As in, entries for older versions of a dependency or dependencies that have since been removed.
`go get` is documented to add dependencies as discovered but I don't think it does any cleanup.

Reply:
@monstermunchkin

Reply:
Yes, you're right. Should be good to go now.